tests/sqllogictests/suites/base/05_ddl/05_0031_internal_column

```diff
@@ -0,0 +1,51 @@
+statement ok
+USE default
+
+statement ok
+DROP TABLE IF EXISTS `05_0031_t`
+
+statement ok
+CREATE TABLE `05_0031_t`(a int)
+
+statement error 1110
+ALTER TABLE `05_0031_t` ADD COLUMN _row_id float
+
+statement error 1110
+CREATE TABLE `05_0031_t_1`(_row_id int)
+
+statement ok
+INSERT INTO TABLE `05_0031_t` values(1),(2)
+
+statement ok
+INSERT INTO TABLE `05_0031_t` values(3)
+
+query II
+SELECT a,_row_id FROM `05_0031_t` order by _row_id
```

Comment:
1. Need more tests to cover other virtual columns.
2. Need more tests to cover  more segments and blocks ( you can insert with numbers  table multiple times to simulate)